tests/test_api.py

```diff
@@ -17,6 +17,7 @@ def test_medication_order(client, mocker, med_order_bundle):
           "subject:Patient.name.given": "Padme",
           "subject:Patient.name.family": "Amidfla",
           "subject:Patient.birthdate": "eq1945-01-15",
+          "dea": "abc123"
```

Comment:
Just to match the typical form of DEA numbers I've seen (caps letters, followed by numbers)
```suggestion
          "dea": "ABC123"
```